routes/agreement-1/agreement-1-fr.njk

```diff
@@ -25,7 +25,7 @@
     {% if data.is_with_partner == "Yes" %}
       We are working with {{ defaultValue("partner_department_name_fr") }}  ({{ defaultValue("partner_department_acronym_fr") }}) to {{ defaultValue("research_goal_fr") }}.
     {% else %}
-      We are conducting this research today to help us {{ data.research_goal_fr }}.
+      We are conducting this research today to help us {{ defaultValue("research_goal_fr") }}.
```

Comment:
It took me forever to realize `defaultValue` was missing from the field I was using for testing...